Solutions/42Crunch API Protection/Data/Solution_42CrunchAPIProtection.json

```diff
@@ -1,7 +1,7 @@
 {
   "Name": "42Crunch API Protection",
   "Author": "42Crunch - plugins@42crunch.com",
-  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/42CrunchLogo.svg\"width=\"75px\"height=\"75px\">",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/42Crunch/azure-sentinel-integration/main/logo/42CrunchLogo.svg\"width=\"75px\"height=\"75px\">",
```

Comment:
Hi @colindomoney-42crunch please use the correct path for logo

Reply:
This is fixed, I had an old version of the solution file in my workspace.